packages/oas/src/analyzer/types.ts

```diff
@@ -33,6 +33,7 @@ export interface OASAnalysis {
      */
     raw_body?: OASAnalysisFeature;
     'x-default': OASAnalysisFeature;
+    'x-readme-ref-name': OASAnalysisFeature;
```

Comment:
mind adding a JS doc for this since this isn't a public extension?

Reply:
Done! Let me know if you would word it differently

Reply:
that's great, thank you!